llama-index-integrations/vector_stores/llama-index-vector-stores-duckdb/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+# Feel free to un-skip examples, and experimental, you will just need to
+# work through many typos (--write-changes and --interactive will help)
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.mypy]
+disallow_untyped_defs = true
+# Remove venv skip when integrated with pre-commit
+exclude = ["_static", "build", "examples", "notebooks", "venv"]
+ignore_missing_imports = true
+python_version = "3.8"
+
+[tool.poetry]
+name = "llama-index-vector-stores-duckdb"
+version = "0.1.0"
+description = "llama-index vector_stores duckdb integration"
+authors = ["Adithya Krishnan <me@krishadi.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "llama_index/"}]
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<3.12"
```

Comment:
We haven't yet fixed the `new-package` tool, but please change this `">=3.8.1<4.0"`

Reply:
fixed this.